### DISCUSSION TITLE
Using cache (redis) makes directus unusable slow

### DISCUSSION THREAD

**Discussion by mvhirsch** [directus/directus#2865] [Archive]
I've a big setup (using a MySQL storing thousands of entries in multiple tables, in-memory usage footprint about 3-4g) and tried to make directus faster. Using directus is fun, using it on kubernetes even more (scaling up to 5 pods at once). So I wanted to add a noticeable cache: using redis (since I'm using mutliple pods at once).

After activating it, responses for a simple "language"-table (like 15 entries?) needed up to 20s (seconds!) instead of 500ms. This is silly. Querying a strongly used table on MySQL (with over 50k entries) takes about 500ms, with activated redis it runs over 60 seconds (which results in a gateway timeout configured in my cluster ...).

What should be cached by using "DIRECTUS_CACHE_*"? Results sets? Schemas sets?
Why is it slower, than faster? Using redis (in-memory, lru, limited to 200mb atm) should make it noticeable faster, shouldn't it? Any idea?

**Comment by benhaynes** [comment]
This is for v8?

**Reply by rijkvanzanten** [reply]
DIRECTUS_CACHE is the v8 naming and this is the directus/directus repo, so I think it's safe to assume this is about v8

**Reply by mvhirsch** [reply]
Correct, I'm using `directus/directus:v8.8.0-apache`
